src/olympia/reviewers/models.py

```diff
@@ -692,6 +692,7 @@ class ReviewActionReason(ModelBase):
     )
     name = models.CharField(max_length=255)
     canned_response = models.TextField(blank=True)
+    canned_block_reason = models.TextField(blank=True)
```

Comment:
missing `null=True` since it's a new field ? Also in the migration

Reply:
For models that can get new rows inserted during the push window that wouldn't do it (the database default isn't kept after the migration is done, so there is a potential window of time where we have insertions without a default, hence why we typically use `null=True`) but for this specific case it's fine yes.